### ISSUE TITLE
Double quotes within verbatim C# strings ends the string

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [PrismJS/prism#582]
For example:

````csharp
string quote = @"Luis: ""Patrick, where did you get that overnight bag?""
Patrick: ""Jean Paul Gaultier.""";

System.Console.WriteLine(quote);
````

It looks like Prism isn't correctly detecting that the quote after "bag?" doesn't end the string? Actual usage is here: http://highschoolhacking.club/CSharp/Strings#Verbatim-Strings

**Comment by Golmote** [comment]
Thanks for reporting. This should be fixed now (d795397ef0880bfb79efa2481b7155981be8ef4e).

**Comment by JoshuaKGoldberg** [comment]
That was a fantastic turnaround time, thanks so much!
